frontend/src/components/review/WIPComment.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useContext, useState } from "react";
 import { useMutation } from "@apollo/client";
-import { Flex, Button, Textarea } from "@chakra-ui/react";
```

Comment:
The changes in the file are minor UI fixes and putting arguments in alphabetical order.
UI fixes:
- Added "Replying to ..."
- Changed placeholder to "Add a comment"
- Fixed margins to match Figma